INSTALL.md

```diff
@@ -1,115 +1,226 @@
+
+```bash
+conda update conda
+```
+
+Finally, restart your shell or reload your profile/rc file to ensure your `$PATH` variable was updated:
+
+````bash
+source ~/.bash_profile
+````
+
+## Setup miniconda environment for QIITA
+
+To ensure that QIITA can install its dependencies – some of which require older or specific distributions of pip packages – without interfering with any pre-existing, globally installed pip packages you might have, you will setup a virtual environment in conda that you must *activate* whenever you are working with or running QIITA.
+
+```bash
+conda create --yes --name qiita python=2.7 pip nose flake8 pyzmq networkx pyparsing natsort mock future libgfortran seaborn 'pandas>=0.18' 'matplotlib>=1.1.0' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1' --channel https://conda.anaconda.org/OpenMDAO
+```
+
+If you receive an error message about conda being unable to find one of the specified packages in its repository, you will have to manually search for them (see troubleshooting miniconda below).
+
+Next, activate your newly created virtual environment for qiita:
+
+```bash
+source activate qiita
+```
+
+(When you want to deactivate this environment, e.g. to return to a different project or back to your global python and pip packages, run `source deactivate`)
+
+If your new conda environment is functioning correctly, you should see this kind of output when you run `which python`, indicating that the `python` command now refers to the python binary in your new virtual environment, rather than a previous global default such as `/usr/bin/python`:
+
+```
+▶ which python
+/Users/your_username/miniconda2/envs/qiita/bin/python
+(qiita)
+```
+
+If you don't see this output, your `$PATH` variable was setup incorrectly or you haven't restarted your shell.
+
+As long as you are in the active qiita environment, commands such as `pip install` or `python` will refer to and be contained within this virtual environment.
+
 
 Install the non-python dependencies
 -----------------------------------
 
 * [PostgreSQL](http://www.postgresql.org/download/) (minimum required version 9.3.0, we have tested most extensively with 9.3)
 * [redis-server](http://redis.io) (we have tested most extensively with 2.8.17)
+* [hdf5](https://www.hdfgroup.org/HDF5/)
+
+There are several options to install these dependencies depending on your needs:
+
+- You could install them via conda, however, the anaconda repository may not have the exact versions of these that you want. 
+- You could install the exact versions we tested as per instructions on their websites, and thus make their available globally on your operating system. This may or may not cause problems if other software on your system requires a differing, specific version of the packages. These instructions were tested with this method.
+- You could setup a full development environment with [Vagrant](https://www.vagrantup.com/), and continue using conda under it to primarily manage python dependencies.
```

Comment:
@josenavas: conda appears primarily built to support python use cases. It does not have well maintained or older versions of non-python dependencies that Qiita recommends, such as Postgres 9.3. 

Vagrant is like virtualenv on an operating system level with two advantages:
1. You can provide a vagrant 'box' (basically an image) with a pre-built ubuntu server with the correct versions of hdf5, postgres, etc. pre-built and pre-configured as well as conda with all the Qiita packages already installed. So no rebuilding or system config needed.
2. So it's very easy to download the image and you can launch it by doing `vagrant up` in the Qiita directory, and `vagrant ssh` into the virtual machine. The Qiita directory on your local machine is kept in sync with the one in the virtual machine.
3. Every Qiita developer can theoretically very easily replicate the same OS and conda environment to develop Qiita.